libs/ui/src/display-line/display-line.component.ts

```diff
@@ -37,6 +39,11 @@ export class WizardsDisplayLineComponent {
     return `${this.firstValue} ${this.secondValue}`;
   }
 
+  get shouldDisplayAdditionalValue(): boolean {
+    return this.firstValue !== null && this.firstValue !== undefined && this.firstValue !== '' ||
```

Comment:
Robisz dwa razy to samo. Napisz na to funkcje typu:

valueExist(value: string): boolean {
// tutaj logike
}

i dzieki temu masz
return this.valueExist(firstValue) || this.valueExist(secondValue)